benchmark_utils/transformation.py

```diff
@@ -43,51 +47,58 @@ def channels_dropout(
         The labels.
 
     """
-    transform = ChannelsDropout(probability=probability, random_state=seed)
+
+    seed = gen_seed()
```

Comment:
```suggestion
    rng = np.random.RandomState(seed)
```

Reply:
For a reproducible sequence of random numbers, the best is to use a random number generator, that will be used to sample different number for each draw, but with a reproducible order.